src/MockBot.UnitTests/ChatControllerTests.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MockBot.Api.Controllers;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+using Moq;
+using Xunit;
+
+namespace MockBot.UnitTests
+{
+    public class ChatControllerTests
+    {
+        private readonly ChatController sut;
+        private readonly Mock<IChatService> mockChatService;
```

Comment:
It is not enforced by code styling but normally private vars are prefixed with an underscore. At the very least, let's make it consistent for this file?

Reply:
Will do